libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -240,6 +240,14 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
   @Input({ transform: booleanAttribute })
   public labelHidden = false;
 
+  /**
+   * [Persistent inline help text](https://developer.blackbaud.com/skyux/design/guidelines/user-assistance#inline-help) that provides
+   * additional context to the user.
+   * @preview
+   */
+  @Input()
+  public hintText: string | undefined;
```

Comment:
alphabetical order for input. Also the documetation doesn't make sense as in this scenario the help text is not inline with the checkbox.

Reply:
Inputs are alphabetized by the docs tools automatically. The documentation string is the same as every other `hintText` input, and "inline" refers to the locality of the text, not that it's on the same actual line

Reply:
I'm good with resolving this to match the pattern set elsewhere. I am curious about that "inline" wording, but I don't know enough to know otherwise.